cmd/helm/dependency_update.go

```diff
@@ -43,7 +44,7 @@ in the Chart.yaml file, but (b) at the wrong version.
 `
 
 // newDependencyUpdateCmd creates a new dependency update command.
-func newDependencyUpdateCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
+func newDependencyUpdateCmd(_ *action.Configuration, out io.Writer) *cobra.Command {
```

Comment:
Same here, private function
```suggestion
func newDependencyUpdateCmd(out io.Writer) *cobra.Command {
```